tests/interfaces/test_batch_jax.py

```diff
@@ -47,36 +48,15 @@ def cost(a, device):
                 device,
                 gradient_fn=param_shift,
                 gradient_kwargs={"shift": np.pi / 4},
-                interface="jax",
+                interface=interface,
             )[0][0]
 
         res = jax.grad(cost)(a, device=dev)
 
         for args in spy.call_args_list:
             assert args[1]["shift"] == np.pi / 4
 
-    @pytest.mark.skip(reason="just because")
-    def test_unknown_gradient_fn_error(self):
-        """Test that an error is raised if an unknown gradient function
-        is passed"""
-        a = jnp.array([0.1, 0.2])

```

Comment:
How come this test was removed?

Reply:
It was being skipped with a not too informative reason in the first place (my bad... :sweat_smile: ) and we don't seem to hit such errors with any of the interfaces anymore. Let me know if I'm missing something.

Reply:
Oh! I missed the `skip` decorator. What an uninformative message 😆 Might be worth seeing who committed that, and asking why.

I hope it wasn't me?

Reply:
It was me, unfortunately, in https://github.com/PennyLaneAI/pennylane/pull/1634 :crying_cat_face: If I recall correctly, each interface had a `raise ValueError("Unknown gradient function.")` branch originally when an unknown gradient function was supplied. At the time of writing up the batch execution PR for JAX, this test was hence copied, but the case didn't work as expected for the JAX interface. Raising the error was removed from the other interfaces along with the test cases.

E.g., see the original batch execute PR adding the test case for the Torch interface: 
https://github.com/PennyLaneAI/pennylane/pull/1549

And the PR that removes this test case for Torch, TF and Autograd:
https://github.com/PennyLaneAI/pennylane/pull/1618/

Reply:
Ah, that makes sense! thanks for double checking this Antal